client/src/pages/Login.js

```diff
@@ -0,0 +1,166 @@
+import React, { useState } from "react";
+import Button from "@material-ui/core/Button";
+import TextField from "@material-ui/core/TextField";
+import Container from "@material-ui/core/Container";
+import Paper from "@material-ui/core/Paper";
+import logo from "../assets/logo.png";
+import { makeStyles } from "@material-ui/core/styles";
+import Typography from "@material-ui/core/Typography";
+import Divider from "@material-ui/core/Divider";
+import { Link } from "react-router-dom";
+import googleLogo from "../assets/googlesvg1.svg";
+
+const useStyles = makeStyles(theme => ({
+  root: {
+    display: "flex",
+    justifyContent: "center",
+    flexDirection: "column"
+  },
+  formMain: {
+    marginTop: theme.spacing(1),
+    display: "flex",
+    flexDirection: "column",
+    alignItems: "center",
+    padding: "10%",
+    fontFamily: theme.typography.fontFamily
+  },
+  footer: {
+    fontFamily: theme.typography.fontFamily,
+    padding: "5%",
+    textAlign: "center"
+  },
+  logo: {
+    marginTop: "10vh",
+    marginLeft: "auto",
+    marginRight: "auto",
+    marginBottom: "5vh"
+  },
+  submit: {
+    margin: theme.spacing(3, 0, 2)
+  },
+  button: {
+    background: theme.palette.primary.button,
+    color: "white",
+    padding: "15px 50px 15px 50px",
+    marginTop: "15%",
+    marginBottom: "10%"
+  },
+  link: {
+    marginLeft: 3,
+    color: theme.palette.primary.main,
+    textDecoration: "none"
+  },
+  paper: {
+    width: "80%",
+    margin: "auto"
+  },
+  formInput: {
+    marginTop: "20%",
+    width: "90%"
+  },
+  formLabel: {
+    marginBottom: "-10px",
+    textAlign: "center"
+  }
+}));
+
+const LogInPage = () => {
+  const classes = useStyles();
+  const [email, setEmail] = useState(null);
+  //welcomeMsg is true if use has entered an email and pressed continue button
+  const [welcomeMsg, showWelcome] = useState(false);
+
+  const handleClick = event => {
+    event.preventDefault();
+    //cehck if user has entered an email
+    if (email) {
+      showWelcome(true);
+    }
+  };
+
+  const handleChange = event => {
+    setEmail(event.target.value);
+    console.log(email);
```

Comment:
You should generally remove console.log statements before merging (some exceptions on backend obviously)